examples/test_flow_dir/test_flow_dir.c

```diff
@@ -155,19 +155,27 @@ static int
 packet_handler(struct rte_mbuf *pkt, struct onvm_pkt_meta *meta,
                __attribute__((unused)) struct onvm_nf_local_ctx *nf_local_ctx) {
         static uint32_t counter = 0;
-        struct onvm_flow_entry *flow_entry = NULL;
+        struct onvm_flow_entry *flow_entry;
         int ret;
 
+        if (!onvm_pkt_is_ipv4(pkt)) {
+                meta->action = ONVM_NF_ACTION_DROP;
+                return 0;
+        }
+
         if (++counter == print_delay) {
                 do_stats_display(pkt);
                 counter = 0;
         }
 
-        ret = onvm_flow_dir_get_pkt(pkt, &flow_entry);
+        struct onvm_ft_ipv4_5tuple key;
+        memset(&key, 0, sizeof(struct onvm_ft_ipv4_5tuple));
+        onvm_ft_fill_key(&key, pkt);
+        ret = onvm_ft_lookup_key(sdn_ft, &key, (char **)flow_entry);
```

Comment:
I noticed that using the lookup pkt in the flow director api either resulted in a seg fault or did not perform a correct lookup or add to the table.